main.py

```diff
@@ -0,0 +1,23 @@
+from sanic import Sanic
+from sanic import response
+from utils import perform_notebook_conversion, authorized
+
+
+# Webservice routing
+app = Sanic('douglass')
+app.config.from_pyfile('config.py')
+
+
+@app.get('/')
```

Comment:
`/status` for the status endpoint? not sure what Broad convention is

Reply:
This should indeed by /status. I will fix